lib/loggerator/metrics.rb

```diff
@@ -1,18 +1,16 @@
-require_relative 'loggerator'
+require "loggerator"
 
 module Loggerator
   module Metrics
     include Loggerator
     extend self
 
-    @@metrics_name = 'loggerator'
-
     def name=(name)
-      @@metrics_name = name
+      Loggerator.config.metrics_app_name = name
     end
```

Comment:
How many other people are using this?  If just us, kill with fire.  Otherwise :+1: on deprecation.